strings/barcode_validator.py

```diff
@@ -83,6 +83,6 @@ def get_barcode(barcode: str) -> int:
     barcode = get_barcode(input("Barcode: ").strip())
 
     if is_valid(barcode):
-        print(f"'{barcode}' is a valid Barcode")
+        print(f"'{barcode}' is a valid Barcode.")
```

Comment:
```suggestion
        print(f"'{barcode}' is a valid barcode.")
```
barcode isn't a noun